src/index.ts

```diff
@@ -35,28 +44,33 @@ if (require.main === module) {
 
   log.debug('Starting Auth0 Deploy CLI Tool');
 
-  // Set log level
-  log.transports.console.level = params.level;
```

Comment:
I did make this small deletion, but it's fairly clear that `params.level` isn't getting set anywhere. I think it is a relic of an older refactor. Not worth the noise to keep.